src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -121,18 +123,49 @@ private void seedCourse() {
 
             Random rand = new Random();
 
+            String courseId = UUID.randomUUID().toString();
             try {
                 String courseName = String.format("Course %s", i);
                 String courseInstitute = String.format("Institute %s", i);
                 String courseTimeZone = String.format("Time Zone %s", i);
-                Course course = new Course(UUID.randomUUID().toString(), courseName, courseTimeZone, courseInstitute,
-                        getRandomInstant(),
+                Course course = new Course(courseId, courseName, courseTimeZone, courseInstitute, getRandomInstant(),
                         rand.nextInt(3) > 1 ? null : getRandomInstant(), // set deletedAt randomly at 25% chance
                         false);
                 ofy().save().entities(course).now();
             } catch (Exception e) {
                 log(e.toString());
             }
+
+            seedStudents(i, courseId);
```

Comment:
Should seedStudents be in the try block? such that if there is an exception then seedStudents will not run

Reply:
Makes sense, could abstract course creation to a function and put both of the methods in the try block